src/main/java/uk/gov/companieshouse/filinghistory/api/mapper/DateUtils.java

```diff
@@ -21,4 +22,8 @@ public static String instantToString(final Instant inputDate) {
                 .map(date -> LocalDate.ofInstant(date, ZoneOffset.UTC).toString())
                 .orElse(null);
     }
+
+    public static FilingHistoryDeltaTimestamp makeNewTimeStampObject(final Instant inputTimeStamp, final String contextId) {
```

Comment:
This should be a constructor on the FilingHistoryDeltaTimestamp class if you do not want to use the builder style setters in multiple places. It will also need a default constructor for Mongo integration.

Reply:
ah I forgot to tidy that up, I'm happy to use the builder style setters if you like or a constructor - depends on what you think is better?

Reply:
Setters are fine for me.

Reply:
done